### PR TITLE
Update Poetry smoke test for 1.8.3

### PR DESCRIPTION
**Pull request by edgarrmondragon**
* Blocking https://github.com/dependabot/dependabot-core/pull/9705

### CODE REVIEW

**Review by TunPyaeSone — APPROVED** [review]

**Comment by ulgens** [comment]
@TunPyaeSone @edgarrmondragon Is there anything blocking the merge?

**Comment by edgarrmondragon** [comment]
> @TunPyaeSone @edgarrmondragon Is there anything blocking the merge?

Just a maintainer hitting the merge button really 😅

**Review by jeffwidman — APPROVED** [review]

**Comment by jeffwidman** [comment]
Thanks for putting up this PR and for the friendly nudges. 

I just merged it in core, and rebased here--it should pickup the new version of core and hopefully passes.

**Pull request merged**
